plugins/ua_network_tcp.c

```diff
@@ -42,13 +42,16 @@
 # ifdef __QNX__
 #  include <sys/socket.h>
 # endif
+# ifdef __OpenBSD__
+#  include <sys/socket.h>
+# endif
```

Comment:
Perhaps we could have something [like that](http://nadeausoftware.com/articles/2012/01/c_c_tip_how_use_compiler_predefined_macros_detect_operating_system#BSD)
````
#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
#include <sys/param.h>
#if defined(BSD)
	include<sys/socket.h>

#endif
#endif
````
This would support DragonFly BSD, FreeBSD, OpenBSD, NetBSD and Apple OSX/iOS